frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -110,6 +110,11 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit, OnD
    */
   showLeaderboardUpdate = false;
 
+  /**
+   * Returns true is leaderboard is public for the selected phase
```

Comment:
Change `Returns true is leaderboard is public for the selected phase` to `Returns true if leaderboard is public for the selected phase`